nginx/build_config.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import time
+from urllib import parse, request, error
+import warnings
+
+import jinja2
+import yaml
+import yaml.error
+
+
+# Do not include leading or trailing slashes in these routes
+PROTECTED_ROUTES = ["saml", "auth", "_nginx"]
+ALLOWED_ROUTES = json.loads(os.environ.get('TURNPIKE_ALLOWED_ROUTES', '["public", "api", "app"]'))
```

Comment:
Is the plan to set `TURNPIKE_ALLOWED_ROUTES` in prod to allow for `docs`?

Reply:
Yep!